state/filenames.go

```diff
@@ -15,5 +15,7 @@
 package state
 
 const (
-	ClusterFile = "cluster.yaml"
+	ClusterJsonFile = "cluster.json"
+	ClusterJsonPath = "/etc/kubicorn/"
```

Comment:
It definitely makes more sense, I was going based off of what was in the issue. Since that's more flexible than I realized, I 100% agree.